siac/parse.go

```diff
@@ -65,15 +64,35 @@ func periodUnits(blocks types.BlockHeight) string {
 	return fmt.Sprint(blocks / 1008) // 1008 blocks per week
 }
 
-// parsePeriod converts a number of weeks to a number of blocks.
+// parsePeriod converts a duration specified in blocks, hours, or weeks to a
+// number of blocks.
 func parsePeriod(period string) (string, error) {
-	var weeks float64
-	_, err := fmt.Sscan(period, &weeks)
-	if err != nil {
-		return "", errUnableToParseSize
+	units := []struct {
+		suffix     string
+		multiplier float64
+	}{
+		{"b", 1},        // blocks
+		{"blocks", 1},   // blocks
+		{"h", 6},        // hours
+		{"hours", 6},    // hours
+		{"w", 1008},     // weeks
+		{"weeks", 1008}, // weeks
```

Comment:
probably should add days as well.

Also, should note that these values are dependent on the const in the `types` package, it's probably worth formalizing that in case we ever switch to Jute.